### PR TITLE
(VDB-1594) Add 1.1.0 addresses [staging]

### PR DESCRIPTION
**Pull request by rmulhol**

### CODE REVIEW

File: transformers/shared/constants/method.go [hunk 1]
```diff
@@ -48,6 +48,19 @@ func FlipABI() string {
 		"MCD_FLIP_ZRX_A_1.0.9",
 	})
 }
+func FlipV110ABI() string {
+	return GetABIFromContractsWithMatchingABI([]string{
+		"MCD_FLIP_BAT_A_1.1.0",
+		"MCD_FLIP_ETH_A_1.1.0",
+		"MCD_FLIP_KNC_A_1.1.0",
+		"MCD_FLIP_MANA_A_1.1.0",
+		"MCD_FLIP_TUSD_A_1.1.0",
```
Comment (yaoandrew) [inline transformers/shared/constants/method.go:57]: Do we need USDT and PAXUSD added here or is that in another PR?

**Review by yaoandrew — APPROVED** [review]
Just had a question on the newer collateral additions. Otherwise LGTM. 🚀

File: environments/docker.toml [hunk 2]
```diff
@@ -336,11 +336,13 @@
         repository = "github.com/makerdao/vdb-mcd-transformers"
```
Comment (elizabethengelman) [inline environments/docker.toml:406]: Do we also need to add these new contract versions to the storage transformers list as well?

Comment (elizabethengelman) [reply environments/docker.toml:406]: Also, I think we'd need new storage transformer initializers for those as well.

File: transformers/shared/constants/method.go (hunk 1 above)
Comment (elizabethengelman) [reply transformers/shared/constants/method.go:57]: I am adding USDT and PAXUSD and #316 😄

File: environments/docker.toml (hunk 2 above)
Comment (rmulhol) [reply environments/docker.toml:406]: will be included when #302 and #303 are merged 👍

**Pull request merged**